rocky/rocky/templates/organizations/organization_list.html

```diff
@@ -11,11 +11,20 @@
         <section>
             <div>
                 <h1>{% translate "Organizations" %}</h1>
-                {% if perms.tools.add_organization %}
-                    <div class="horizontal-view toolbar">
+                <div class="horizontal-view toolbar">
+                    {% if perms.tools.can_recalculate_bits %}
+                        <form method="post" class="inline">
+                            {% csrf_token %}
+                            <button type="submit"
+                                    class="dropdown-button"
+                                    name="action"
+                                    value="recalculate">{% translate "Rerun all bits" %}</button>
```

Comment:
```suggestion
                                    value="recalculate">{% translate "Rerun all bits for all my organizations" %}</button>
```

Reply:
Applied. It makes it more ugly though, as the button will not become as wide.